netbox/ipam/tests/test_api.py

```diff
@@ -1152,7 +1156,8 @@ def setUpTestData(cls):
         ]
 
         cls.bulk_update_data = {
-            'policy': vlan_translation_policies[1].pk,
+            'policy': vlan_translation_policies[2].pk,
```

Comment:
Before this change, the test was not producing the expected number of ObjectChange records, which failed a check in the updated test suite.